remote_challenge_evaluation/main.py

```diff
@@ -79,7 +82,7 @@ def update_finished(
             phase_pk = message_body.get("phase_pk")
             # Get submission details -- This will contain the input file URL
             submission = evalai.get_submission_by_pk(submission_pk)
-
+            challenge_phase = evalai.get_challenge_phase_by_pk(phase_pk)
```

Comment:
Why do you need to fetch challenge phase again? can't we access `submission.challenge_phase`?

Reply:
We get a dictionary with `challenge_phase` which actually gives us `challenge_pk`. Since we pass the `codename` to the `evaluate` method, we need to get the challenge phase and then the corresponding codename.